src/renderer/decoder/lineDecoder.js

```diff
@@ -72,7 +80,7 @@ function addLine (lineString, vertices, normals) {
                 // Third triangle
                 addTriangle(
                     [currentPoint, currentPoint, currentPoint],
-                    [[0, 0],
+                    [[0, 1e-37],
```

Comment:
What's that tiny value for?

Reply:
In the shader is used this expression `normal == vec2(0.)` to check if the point belongs to the fill or stroke. The problem is that we have now a stroke vertex with normal (0,0). So adding this tiny value marks the vertex as "stroke vertex" but does not affect the position. Let me add a comment with this information.